packages/core/src/services/documentLogs/computeDocumentLogsWithMetadata.ts

```diff
@@ -1,27 +1,145 @@
-import { and, desc, eq } from 'drizzle-orm'
+import {
+  and,
+  desc,
+  eq,
+  getTableColumns,
+  isNotNull,
+  or,
+  sql,
+  sum,
+} from 'drizzle-orm'
 
 import { Commit } from '../../browser'
 import { database } from '../../client'
 import {
-  createDocumentLogQuery,
-  getCommitFilter,
-} from './_createDocumentLogQuery'
+  DocumentLogsWithErrorsRepository,
+  DocumentLogWithErrorScope,
+} from '../../repositories'
+import { commits, providerLogs } from '../../schema'
+
+function getRepositoryScopes(workspaceId: number, db = database) {
+  const scope = new DocumentLogsWithErrorsRepository(workspaceId, db).scope
+
+  return { scope }
+}
+
+function getCommitFilter(draft?: Commit) {
+  return draft
+    ? or(isNotNull(commits.mergedAt), eq(commits.id, draft.id))
+    : isNotNull(commits.mergedAt)
+}
+
+function getCommonQueryConditions({
+  scope,
+  documentUuid,
+  draft,
+  allowAnyDraft,
+}: {
+  scope: DocumentLogWithErrorScope
+  documentUuid?: string
+  allowAnyDraft?: boolean
+  draft?: Commit
+}) {
+  const byDocumentUuid = documentUuid
+    ? eq(scope.documentUuid, documentUuid)
+    : sql`1 = 1`
```

Comment:
If `documentUuid` is not passed we don't care and use `1 = 1`. This case is covering `/evaluations` playground where we get a `documentLog.resolvedContent` for the evaluation variables

We don't have the scope of `documentUuid`

Reply:
hmm don't get it is this gonna match all document logs a pelo? wouldn't allow it

Reply:
Here
![image](https://github.com/user-attachments/assets/b3de0fb6-e8ae-4b2a-9783-6db76acd37eb)

Reply:
In the playground we can import any document log from any project and any document

Reply:
But we filter by `documentLogUuid` https://github.com/latitude-dev/latitude-llm/blob/main/packages/core/src/services/documentLogs/fetchDocumentLogWithMetadata.ts#L40-L41